src/compiler/checker.ts

```diff
@@ -24563,7 +24563,7 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
     }
 
     function getTypeFromFlowType(flowType: FlowType) {
-        return flowType.flags === 0 ? (flowType as IncompleteType).type : flowType as Type;
+        return flowType.flags === 0 ? flowType.type : flowType as Type;

```

Comment:
This is problematic as we won't compile under 4.8.4 anymore with this change to fix linting. VS Code's nightly has also not updated for some reason.